lib/collection/src/wal_delta.rs

```diff
@@ -24,11 +28,14 @@ pub struct RecoverableWal {
     /// This means two things:
     /// - this WAL has at least all these clock versions (same as `highest_clocks`)
     /// - this WAL cannot resolve any delta below any of these clocks
+    //
+    // TODO: Rename to `oldest_available_records` or `oldest_resolvable_records`?
```

Comment:
Don't really like that name, but I don't have a better suggestion. I'd prefer something short.